patch/0080-TMP-mlxsw-i2c-Prevent-transaction-execution-for-spec.patch

```diff
@@ -0,0 +1,82 @@
+From b041f1a4ed2da36a831548d8539e8869cea33b40 Mon Sep 17 00:00:00 2001
+From: Vadim Pasternak <vadimp@nvidia.com>
+Date: Tue, 31 May 2022 10:27:38 +0300
+Subject: [PATCH v4.9 ISSU WA 1/1] TMP: mlxsw: i2c: Prevent transaction
+ execution for special chip states
+
+Do not run transaction in cases chip is in reset or in-filed update states.
+
```

Comment:
Is that commit upstream? Plesae explain the TMP.

Reply:
it is not and it cannot be upstream. this is a way for bug fix and we wont be able to upstream it.

Reply:
I do not understand. Please elaborate.

Reply:
Upstream mlxsw driver doesn't support ISSU, so we can't submit ISSU related filtering to upstream.

Reply:
Only 4.19(?) or all versions? The policy of SONiC is that long term all patches have to be upstream to lessen the maintenance burden especially when updating to newer Linux kernel releases.

Can you please donate that in the patch description?

Reply:
In the repository, I did `git grep -i issu`, and nothing showed up regarding *ISSU*. Where can I find more information?

Reply:
Within SONiC, ISSU may be called warm reboot or warmboot. Some more details are [here](https://github.com/sonic-net/SONiC/blob/master/doc/warm-reboot/SONiC_Warmboot.md).

Reply:
Thank you: ISSU = In-Service-Software-Upgrade